platform-includes/getting-started-config/javascript.solidstart.mdx

```diff
@@ -0,0 +1,102 @@
+To set up the Sentry SDK initialize it in the client and server.
+
+### Client-side Setup
+
+Initialize the Sentry SDK in your `entry-client.tsx` file.
+
+If you're using Solid Router, add the `solidRouterBrowserTracingIntegration` to collect meaningful performance data about the health of your page loads and associated requests.
+
+<SignInNote />
+
+```jsx {filename:entry-client.tsx}
+import * as Sentry from '@sentry/solidstart';
+import { solidRouterBrowserTracingIntegration } from '@sentry/solidstart/solidrouter';
+import { mount, StartClient } from '@solidjs/start/client';
+
+Sentry.init({
+  dsn: '__PUBLIC_DSN__',
+  integrations: [solidRouterBrowserTracingIntegration()],
+  tracesSampleRate: 1.0, //  Capture 100% of the transactions
+});
+
+mount(() => <StartClient />, document.getElementById('app'));
+```
+
+### Server-side Setup
+
+Create an instrument file `instrument.server.mjs` and initialize the Sentry SDK.
+
+<SignInNote />
+
+```javascript {filename:instrument.server.mjs}
+import * as Sentry from '@sentry/solidstart';
+
+Sentry.init({
+  dsn: '__PUBLIC_DSN__',
+  tracesSampleRate: 1.0, //  Capture 100% of the transactions
+});
+```
+
+### Instrumentation
+
+Complete the setup by adding the Sentry middleware to your `middleware.ts` file. If you don't have a `middleware.ts` file yet, create one:
```

Comment:
You could link to the [middleware docs](https://docs.solidjs.com/solid-start/advanced/middleware) here (for people who don't have this file yet, this might be useful).

M: Also interesting here: Where should this be located

Reply:
Location again depends on the project structure. I could default it to `src` but feel like it doesn't really add any extra info.

Reply:
Rewrote this section a bit, added the link and location of the middleware file.